users/views.py

```diff
@@ -92,30 +93,13 @@ def index(request: HttpRequest) -> HttpResponse:
     return render(request, "home.html", context={"num_visits": num_visits})
 
 
-class SignUp(APIView):
-    serializer_class = CustomRegisterSerializer
-    renderer_classes = [renderers.TemplateHTMLRenderer]
+class SignUp(FormView):
+    form_class = CustomUserSignUpForm
     template_name = "signup.html"
 
-    @classmethod
-    def get(cls, request: HttpRequest) -> Response:
-        serializer = CustomRegisterSerializer(context={"request": request})
-        return Response({"serializer": serializer})
-
-    @classmethod
-    def post(cls, request: HttpRequest) -> Union[Response, HttpResponseRedirectBase]:
-        serializer = CustomRegisterSerializer(data=request.data)
-        if not serializer.is_valid():
-            return Response(
-                {
-                    "serializer": serializer,
-                    "errors": serializer.errors,
-                    "non_field_errors": serializer.errors.get("non_field_errors"),
-                }
-            )
-        else:
-            serializer.save(request)
-            return redirect("login")
+    def form_valid(self, form: CustomUserSignUpForm) -> Union[Response, HttpResponseRedirectBase]:
+        form.save()
+        return redirect("home")
```

Comment:
Maybe we should add some other view here saying that account was created and you can login now? @Karrp @kbeker  what do you think?

Reply:
So I am waiting for answer from them.

Reply:
I think there should be some success info.
Other view should be ok.
Or maybe popup would be better? And redirect after few seconds or pressing "OK" button.

Reply:
I think that popup with this info would be nice :)